modules/compute/disk-encryption-set/main.bicep

```diff
@@ -91,7 +90,7 @@ resource keyVault 'Microsoft.KeyVault/vaults@2021-10-01' existing = {
 }
 
 // Note: This is only enabled for user-assigned identities as the service's system-assigned identity isn't available during its initial deployment
-module keyVaultPermissions 'modules/nested_keyVaultPermissions.bicep' = [for (userAssignedIdentityId, index) in items(userAssignedIdentities): {
+module keyVaultPermissions 'modules/nested_keyVaultPermissions.bicep' = [for (userAssignedIdentityId, index) in items(formattedUserAssignedIdentities): {
```

Comment:
Maybe it would be easier to directly reference the user assigned identities. For example via
`[for (userAssignedIdentityResourceId, index) in (managedIdentities.?userAssignedResourcesIds ?? []): {`

and update also the reference inside the loop to simply `userAssignedIdentityResourceId` instead of the `.key` reference.

Reply:
Implemented as suggested: https://github.com/Azure/ResourceModules/pull/4124/commits/239d045c98ac99e4921240dde8f69e40077ea4a0